jinja2/tests.py

```diff
@@ -147,28 +125,69 @@ def test_iterable(value):
     return True
 
 
+def test_in(value, seq):
+    """Check if value is in seq.
+
+    .. versionadded:: 2.10
+    """
+    return value in seq
+
+
 def test_escaped(value):
     """Check if the value is escaped."""
     return hasattr(value, '__html__')
 
 
-def test_greaterthan(value, other):
-    """Check if value is greater than other."""
+def test_compare_eq(value, other):
+    """Check if an object has the same value as another object:
+
+    .. sourcecode:: jinja
+
+        {% if foo.expression is equalto 42 %}
+            the foo attribute evaluates to the constant 42
+        {% endif %}
+
+    This appears to be a useless test as it does exactly the same as the
+    ``==`` operator, but it can be useful when used together with the
+    `selectattr` function:
+
+    .. sourcecode:: jinja
+
+        {{ users|selectattr("email", "equalto", "foo@bar.invalid") }}
+
+    .. versionadded:: 2.8
+    """
+    return value == other
+
+
+def test_compare_ne(value, other):
+    """Check if an object is not equal to another object."""
+    return value != other
+
+
+def test_compare_gt(value, other):
+    """Check if an object is greater than another object."""
     return value > other
 
 
-def test_lessthan(value, other):
-    """Check if value is less than other."""
+def test_compare_ge(value, other):
+    """Check if an object is greater than or equal to another object."""
+    return value >= other
+
+
+def test_compare_lt(value, other):
+    """Check if an object is less than another object."""
     return value < other
 
 
-def test_in(value, seq):
-    """Check if value is in seq.
+def test_compare_le(value, other):
+    """Check if an object is less than or equal to another object."""
+    return value <= other
 
-    .. versionadded:: 2.10
-    """
-    return value in seq
 
+test_equalto = test_compare_eq
+test_greaterthan = test_compare_gt
+test_lessthan = test_compare_lt
```

Comment:
I could make them functions that raise deprecation warnings, but I'm not that concerned. I'll just add a comment to the source.